### PR TITLE
Modify S3336: Change text to LaYC (APPSEC-1217)

### PR DESCRIPTION
**Pull request by gaetan-ferry-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

**Review by loris-s-sonarsource — CHANGES_REQUESTED** [review]
Mostly layout comments

File: rules/S3336/php/rule.adoc [hunk 1]
```diff
@@ -1,31 +1,51 @@
...
-* see the URL over the user's shoulder
-* be sent the URL by the user
-* retrieve the URL from browser history
-* ...
+=== What is the potential impact?
 
-For that reason, it's better to practice a little "tough love" with your users and force them to turn on cookies.
+Attackers with access to any of those disclosure locations will be able to see
+and steal a victim's session token. They can then use it to log as the user,
+impersonate their account, and take advantage of their privileges.
```
[the remark this answers is not in the document]
Comment (gaetan-ferry-sonarsource) [reply rules/S3336/php/rule.adoc:17]: I am not too sure about the general structure of your suggestion. I always found the introduction sentence "Below are some real-world scenarios that illustrate some impacts of an attacker exploiting the vulnerability" to be very artificial and not very fluid.

File: rules/S3336/php/rule.adoc [hunk 2]
```diff
@@ -1,31 +1,51 @@
...
 
+== How to fix it
 
-Since ``++session.use_trans_sid++`` is off by default, this rule raises an issue when it is explicitly enabled.
+Disable the `session.use_trans_sid` setting in the main PHP configuration file.
 
+Note that this parameter is off by default.
```
[the remark this answers is not in the document]
Comment (gaetan-ferry-sonarsource) [reply rules/S3336/php/rule.adoc:23]: That's not really how I like it done but I guess I can change, especially for this kind of issue.

**Review by loris-s-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**